src/guides/v2.3/get-started/gs-curl.md

```diff
@@ -28,6 +28,119 @@ Option | Description
 `-T` | Transfers the specified local file to the remote URL.
 `-X` | Specifies the request method to use when communicating with the HTTP server. The specified request method is used instead of the default GET method.
 
+### Using cUrl in Magento
+
+Magento provides native functionality for cURL instead of using default php cURL. The class ``Magento\Framework\HTTP\Client\Curl`` can be used to work with HTTP protocol using curl library.
+At first, the instance of ``Magento\Framework\HTTP\Client\Curl`` should be created.
+
+```php
+/**
+* Constructor.
+*
+* @param Magento\Framework\HTTP\Client\Curl $curl
+*/
+public function __construct(
+   Magento\Framework\HTTP\Client\Curl $curl
+) {
+   $this->_curl = $curl;
+}
+```
+#### Make GET request using cURL
+
+```php
+// get method
+$this->_curl->get($url);
+
+// output of curl request
+$result = $this->_curl->getBody();
+```
+
+Where the ``$url`` is the endpoint url
+
+#### Make POST request using cURL
+
+```php
+// post method
+$this->_curl->post($url, $params);
+
+// output of curl requestt
+$result = $this->_curl->getBody();
+```
+
+Where the ``$url`` is the endpoint url, ``$params`` an array, the extra parameters can be added in the url.
```

Comment:
We may extend the `$params` argument description. Instead of saying that it's just an "array" I would say that it's an array of data that is being sent via the POST request.